src/component/Stars/Stars.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useRef } from 'react';
+
+import styles from './Stars.module.css';
+
+interface StarsProps {
+  getRating?: (evt: any) => void,
+  rating?: string
+}
+
+const Stars: React.FC<StarsProps> = ({ getRating, rating }) => {
+  const value1 = useRef<HTMLInputElement>(null);
```

Comment:
At first glance this looks like a test code.
Try to avoid generic names. https://martinfowler.com/bliki/TwoHardThings.html